course_info/templates/course_info/index.html

```diff
@@ -45,6 +45,7 @@
     <script>
       window.globals.STATIC_URL = '{% settings_value "STATIC_URL" %}';
       window.schoolOptions = {{ schools | safe }};
+      window.roles = {{ canvas_roles | safe }};
```

Comment:
This is not just the list of id's, it is a list of dicts that contain id and role label. I can change this to say user_roles instead of canvas_roles.